include/psa/crypto_struct.h

```diff
@@ -223,6 +223,29 @@ static inline struct psa_key_derivation_s psa_key_derivation_operation_init(
     return v;
 }
 
+struct psa_key_generation_method_s {
+    /* Future versions may add other fields in this structure. */
+    uint32_t flags;
```

Comment:
Unlike most other structures, I made the fields public rather than add accessors. For `data`, I wanted to allow users to directly fill the array, rather than have to call a function which would copy from some other bit of memory. Then, if `data` is public, why not make `flags` public as well.